skimage/feature/blob.py

```diff
@@ -308,15 +345,21 @@ def blob_log(image, min_sigma=1, max_sigma=50, num_sigma=10, threshold=.2,
         If set intermediate values of standard deviations are interpolated
         using a logarithmic scale to the base `10`. If not, linear
         interpolation is used.
+    exclude_border : int or bool, optional
+        If nonzero int, `exclude_border` excludes blobs from
+        within `exclude_border`-pixels of the border of the image.
```

Comment:
If you want to add this in this PR, could you add a test for it?

Reply:
Test added in bcee760